CHANGELOG.rst

```diff
@@ -5,6 +5,45 @@ Kubernetes Collection Release Notes
 .. contents:: Topics
 
 
+v3.0.0
```

Comment:
I think this needs to be regenerated to get the updates from the changelog.yaml file?